tests/serialize_test.py

```diff
@@ -210,14 +230,18 @@ def make_bomb(depth: int) -> SExp:
         self.assertEqual(len(b10_2), 75)
 
         bomb_20 = make_bomb(20)
-        b20_1 = bomb_20.as_bin(allow_backrefs=False)
+        with pytest.raises(ValueError, match="SExp exceeds maximum size"):
+            # b20_1 = bomb_20.as_bin(allow_backrefs=False)
+            bomb_20.as_bin(allow_backrefs=False)
         b20_2 = bomb_20.as_bin(allow_backrefs=True)
-        self.assertEqual(len(b20_1), 48234495)
+        # self.assertEqual(len(b20_1), 48234495)
         self.assertEqual(len(b20_2), 105)
 
         bomb_30 = make_bomb(30)
         # do not uncomment the next line unless you want to run out of memory
-        # b30_1 = bomb_30.as_bin(allow_backrefs=False)
+        with pytest.raises(ValueError, match="SExp exceeds maximum size"):
+            # b30_1 = bomb_30.as_bin(allow_backrefs=False)
```

Comment:
I think you can remove this commented-out line now, since you have a copy of it below, that throws